src/wpt/harness.js

```diff
@@ -0,0 +1,41 @@
+import { strictEqual } from 'node:assert';
```

Comment:
Side note... let's try not to forget adding the copyright headers to these (speaking to myself here mostly)

Reply:
@npaun ... can you go through and add the necessary copyright headers to the new files?

Reply:
WPT's copyright?

Reply:
Workerd.